lerna-package-lock.json

```diff
@@ -5547,6 +5687,76 @@
 				"nconf": "^0.11.4",
 				"proper-lockfile": "^4.1.2",
 				"semver": "^7.3.4"
+			},
+			"dependencies": {
```

Comment:
Just checked and these sections are fine. For packages that are "outside of the monorepo structure" (like in this case `@fluidframework/test-version-utils-previous`, the previous version of test-version-utils, used for type tests, or `@fluidframework/routerlicious-urlresolver-previous`, similar sceneario) which still depend on different versions of some packages that are not compatible with the versions we're hoisting to the root node_modules (in this case, `nconf@^0.11.4` which depends on `async@1.5.2`, not semver-compatible with `async@^3.2.2`), package lock will have this extra `dependencies` sections to describe packages that need to be installed not in the root node_modules, but directly in the node_modules of the package that has these specific version dependencies.

Reply:
I was worried if `npm i` had created unnecessary changes. I'm glad that it wasn't the case 😄 . Thank you for your review!